.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Basic build test
+
+name: build
+
+on: [pull_request, push, workflow_dispatch]
+
+jobs:
+  build-test:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with: {fetch-depth: 50, submodules: recursive}
+
+      - name: install cmake
+        run: sudo apt update
+      - run: sudo apt install -y cmake
```

Comment:
Normally we build in the container though which has this for us, this is outside the container, shouldn't we pull it manually?

Reply:
.github/workflows/unit-tests.yml doesn't build in the container and seems to be doing fine. Github has a list of default packages somewhere but it's probably faster to just try :-) CMake is really the standard so it would be very surprising not to include it.

(Toolchain is of course the big reason why we need containers)